jsk_robot_common/jsk_robot_startup/CMakeLists.txt

```diff
@@ -29,7 +29,11 @@ find_package(catkin REQUIRED COMPONENTS
   urdf
 )
 
-find_package(Boost REQUIRED)
+find_package(Boost REQUIRED COMPONENTS program_options thread system)
+if ("${BOOST_MAJOR_VERSION}" VERSION_EQUAL "1" AND "${Boost_MINOR_VERSION}" VERSION_GREATER "46")
```

Comment:
When major version of boost is upgraded to 2 or upper, this won't work correctly.

Reply:
Who uses boost upper 2.0?
REP 3 does not support Boost 2.0 https://www.ros.org/reps/rep-0003.html#melodic-morenia-may-2018-may-2023

Reply:
I know, but it doesn't mean jsk_robot won't support boost>=2.0 __forever__.
Also, that's not the point.
You dislike this long boolean comparison line, right?
```
if ("${Boost_MAJOR_VERSION}.${Boost_MINOR_VERSION}.${Boost_SUBMINOR_VERSION}" VERSION_EQUAL "1.47.0" OR
    "${Boost_MAJOR_VERSION}.${Boost_MINOR_VERSION}.${Boost_SUBMINOR_VERSION}" VERSION_GREATER "1.47.0")
```
Actually I would like to use `VERSION_GREATER_EQUAL`, but it can only be available in cmake>=3.7
So I wrote a logical alternative like this.

Reply:
I don't get why you need to concatenate all version numbers into one.
You can compare these numbers separately.
Also, We haven't tested on Boost>=2.0,  so `BOOST_MAJOR_VERSION` should be `1`.

Reply:
OK, I removed `Revert "use chrono in boost >= 1.47.0"` commit.